scripts/sct_label_vertebrae.py

```diff
@@ -849,7 +849,7 @@ def label_segmentation(fname_seg, list_disc_z, list_disc_value, fig_anat_straigh
     """
 
     if fig_anat_straight is None:
-        fig_anat_straight = Param.fig_anat_straight
+        fig_anat_straight = Param().fig_anat_straight
```

Comment:
I have put back the __init__ in param so we do need it. Using class variable would have had to many possible side effects.